Speech_Generator.py

```diff
@@ -38,9 +46,14 @@ def __init__(self, credentials:str="auth.json",
         if (not(os.path.isdir(self.data_dir_path))):
             os.makedirs(self.data_dir_path)
 
-    def read_csv(self, csv_path: str) -> set:
+        self.file_mapping = {
+            "<<GENERICS>>":set()
+        }
+
+    def parse_rows(self, paths:list) -> dict:

```

Comment:
`set` here as well, although I don't remember if the typing declaration can actually throw errors during compile/runtime. Or maybe dict handles the set type as well? Not sure.

Otherwise, I've read through the rest of the logic and it all looks sound to me, awesome stuff!

Reply:
Ooo good point. I will fix that. I believe those annotations are strictly for readability. I don't believe they affect anything at runtime.

Reply:
Fixed the styling error.